nutils/export.py

```diff
@@ -47,34 +47,40 @@ def plotlines_(ax, xy, lines, **kwargs):
     return lc
 
 
-def triplot(name, points, values=None, *, tri=None, hull=None, cmap=None, clim=None, linewidth=.1, linecolor='k', plabel=None, vlabel=None):
+def _triplot(fig, points, values=None, *, tri=None, hull=None, cmap=None, clim=None, linewidth=.1, linecolor='k', plabel=None, vlabel=None):
```

Comment:
It looks like `fig` is only used for creating axes and colorbars. What do you think of changing `fig` into `ax` and creating a colorbar using

```python
cax = matplotlib.colorbar.make_axes_gridspec(ax)
cb = cbar.Colorbar(cax, im, label=vlabel)
```

Reply:
Changed fig to ax and moved colorbar out of _triplot.